### PR TITLE
subsort data, submodule for all d2-data

### PR DESCRIPTION
**Pull request by delphiactual**
Sort the data directory between D1 and D2.
import season, event, and source data automatically from d2-additional-info-module

Also thought about renaming the d1 manifests, but just putting them in their own directory should suffice as adequate separation.

Added a search exclusion for the d1-manifests as searching for "traction" was crashing vscode.

### CODE REVIEW

**Comment by delphiactual** [comment]
need to run `grunt sync`  on during the build process as well

File: .vscode/settings.json [hunk 1]
```diff
@@ -6,6 +6,9 @@
   "editor.formatOnSave": true,
   "eslint.packageManager": "yarn",
   "eslint.validate": ["javascript", "javascriptreact", "typescript", "typescriptreact"],
+  "search.exclude": {
+    "**/data/d1/manifests": true
```
Comment (bhollis) [inline .vscode/settings.json]: Nice.

File: .gitmodules [hunk 2]
```diff
@@ -1,3 +1,6 @@
 [submodule "destiny-icons"]
 	path = destiny-icons
 	url = https://github.com/justrealmilk/destiny-icons
+[submodule "d2-additional-info-module"]
+	path = d2-additional-info-module
```
Comment (bhollis) [inline .gitmodules]: Could you just submodule this directly into the data directory?

File: config/webpack.js [hunk 3]
```diff
@@ -430,7 +430,7 @@ module.exports = (env) => {
           /extension-dist/,
           /\.map$/,
           // Ignore both the webapp manifest and the d1-manifest files
-          /data\/d1-manifest/,
+          /data\/d1\/manifests/,
```
Comment (bhollis) [inline config/webpack.js]: There's also a line in the `CopyWebpackPlugin` config that excludes `missing_sources.json` - can you check that that's still being excluded correctly?

File: .gitmodules (hunk 2 above)
Comment (delphiactual) [reply .gitmodules]: :+1:

**Pull request merged**